Source/Hurl.RulesetManager/MainWindow.xaml

```diff
@@ -0,0 +1,130 @@
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition />
+            <ColumnDefinition Width="2*" />
+            <ColumnDefinition />
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition />
+        </Grid.RowDefinitions>
+
+        <ui:TitleBar Title="Ruleset Manager - Hurl" Grid.ColumnSpan="3" />
+
+        <StackPanel Grid.Row="1" Grid.Column="1">
+            <TextBlock
+                Margin="0,0,0,8"
+                HorizontalAlignment="Center"
+                VerticalAlignment="Center"
+                FontSize="20"
+                Text="Rulesets" />
+            <StackPanel HorizontalAlignment="Center" Orientation="Horizontal">
+                <ui:Button
+                    Width="2in"
+                    Margin="0,0,4,4"
+                    HorizontalAlignment="Stretch">
+                    Create
+                </ui:Button>
+
+                <ui:Button
+                    Width="2in"
+                    Margin="4,0,0,4"
+                    HorizontalAlignment="Stretch">
+                    Test
+                </ui:Button>
+            </StackPanel>
+        </StackPanel>
+
+        <StackPanel
+            Grid.Row="2"
+            Grid.Column="1"
+            VerticalAlignment="Stretch">
+            <Separator Margin="0,4" />
+
+            <StackPanel>
+                <ItemsControl ItemsSource="{Binding}">
+                    <ItemsControl.ItemTemplate>
+                        <DataTemplate>
+                            <ui:CardExpander Margin="0,2" Header="{Binding BrowserName}">
+                                <StackPanel>
+                                    <Grid Margin="0,0,0,8">
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="Auto" />
+                                            <ColumnDefinition />
+                                        </Grid.ColumnDefinitions>
+
+                                        <ui:Button
+                                            Margin="0,0,8,0"
+                                            Padding="12,6"
+                                            Click="Button_Click"
+                                            Icon="{ui:SymbolIcon Edit16}"
+                                            Tag="{Binding Id}">
+                                            Edit
+                                        </ui:Button>
+
+                                        <StackPanel
+                                            Grid.Column="1"
+                                            HorizontalAlignment="Right"
+                                            Orientation="Horizontal">
+                                            <ui:Button
+                                                Margin="0,0,8,0"
+                                                Padding="12,7"
```

Comment:
The button above here has `12,6`, is this intentionally different?

Reply:
Yes, Above btn has text with icon, while this one only has icon.

For some reason, They look differently sized with same padding